### PR TITLE
Update video player loading screen

### PR DESCRIPTION
**Pull request by seanyesmunt**
### Summary

The goal of this is to add a slightly better-looking video player loading screen and set a folder tree structure for internal components. In this PR I have created a folder called `internal` inside of the `video` component folder. Currently, the components are separated by name into folders, but they only have a single `.jsx` file for the main component, as well as any other internal components needed to build the main component.

This leads to huge React files with many components which can make it hard to follow the component tree. I think we should set a style now and try to adhere to it as components grow.

The structure added in this PR is:
```
component/
  ...
  video/
      internal/
          some-internal-comp.jsx // some component that is imported by view.jsx
      view.jsx // the base component
      index.js // connects base component to store

  ...
```

### How it looks

loading

### CODE REVIEW

**Comment by 6ea86b96** [comment]
:+1: from me. I'd love us to break the components up more like this.

File: ui/js/component/video/internal/loading-screen.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const LoadingScreen = ({ status }) =>
+  <div className="video--loading--screen">
```
Comment (kauffj) [inline ui/js/component/video/internal/loading-screen.jsx]: We're using [BEM](http://getbem.com/) as our CSS naming pattern. If our top level block here is `video`, then these class names should be something like:

video__loading-screen
video__loading-screen-content
video__loading-status

`--` should only be used as a modifier, e.g. `video__loading-status--error`.

File: ui/scss/component/_video.scss [hunk 2]
```diff
@@ -28,6 +27,69 @@ video {
...
     /*background: none;*/
   }
+
+  .video--loading--screen {
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .loading--screen--content {
```
Comment (kauffj) [inline ui/scss/component/_video.scss]: Since BEM generates unique class names, we don't typically nest CSS styles, particularly classes.
